header.go

```diff
@@ -3011,12 +3011,20 @@ func (h *ResponseHeader) parseHeaders(buf []byte) (int, error) {
 			return 0, fmt.Errorf("invalid header key %q", s.key)
 		}
 
+		spaceIncluded := false
 		for _, ch := range s.key {
 			if !validHeaderFieldByte(ch) {
+				if ch == ' ' {
+					spaceIncluded = true
+					break
```

Comment:
Yes, you're right. I didn't think that...

Reply:
Changed! Thank you!

Reply:
Wait a minute... Come to think of it, I'm going to ignore it anyway, but do we still need validation? If we need validation of the header key, don't we need validation of the header value?

Reply:
I think there are two choices, one is to validate both the key and the value because if there is the wrong byte, it should fail, and the other is not to do it after have checked the space. And wouldn't it be right not to validate it for fasthttp? I think it would be better to skip the minor verification.

Reply:
In `net/http`, they handle headers [even if they contain spaces](https://cs.opensource.google/go/go/+/master:src/net/textproto/reader.go;l=749-765;drc=fafd4477f3d19f2c11a628e6e407ecf9924309c1). I just realized that you directly ignore this header field. Although I think we should follow the behavior with `net/http` instead of skipping this header. 

We can ask @erikdubbelboer for his opinion.

Reply:
I wanted to follow that action if possible, but net/http doesn't canonicalize the header afterwards, but fasthttp normalizes it through disableNormalizing, so I decided to just ignore it. This is also because it is a common behavior for Internet browsers. If you have any ideas, please let me know.

Reply:
I also prefer it if we follow the behavior of net/http as that is really well tested.